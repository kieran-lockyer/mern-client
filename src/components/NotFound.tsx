import React from "react";
import styled from "styled-components";

const Error = () => {
  return (
    <Container>
      <h1>Error 404</h1>
      <p>The page you are looking for could not be found.</p>
    </Container>
  );
};

const Container = styled.div`
  flex: 1 1 100%;
  padding: 30px 0;
  display: flex;
  flex-direction: column;
  align-items: center;
`;

export default Error;
