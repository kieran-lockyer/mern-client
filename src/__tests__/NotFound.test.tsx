import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../components/NotFound';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

describe('<NotFound />', () => {
  describe('render()', () => {
    test('renders the not found component', () => {
      const wrapper = shallow(<NotFound />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});