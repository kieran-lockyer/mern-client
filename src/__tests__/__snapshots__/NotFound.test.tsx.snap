// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> render() renders the not found component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  flex: 1 1 100%;
  padding: 30px 0;
  display: flex;
  flex-direction: column;
  align-items: center;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <h1>
    Error 404
  </h1>
  <p>
    The page you are looking for could not be found.
  </p>
</StyledComponent>
`;
